docs-src/modules/getting-started/pages/index.adoc

```diff
@@ -1,14 +1,91 @@
 = Getting Started
 
-Kaskada's query language xref:overview:what-is-kaskada.adoc[focuses on event-based data processing] making it easier to reason about when things happen, the state at specific points in time, and how results change over time.
-Kaskada queries are simple, declarative and composable providing a more expressive way to query event-based data.
+== Reasoning about events with timelines
 
-There are several ways to quickly get started with Kaskada.
+Kaskada is a unified event processing engine that provides stateful stream processing in a high-level query language designed specifically for reasoning about events in bulk and in real time.
+Kaskada is built on the idea of a _timeline_ - the history of how a value changes over time for a specific entity or group.
```

Comment:
Use of "history" here could be confusing with later "timeline history" and "timeline snapshot"... but maybe not, since the "timeline history" there is the history of how a value changes over time...